src/mmw/apps/modeling/mapshed/tasks.py

```diff
@@ -244,14 +260,51 @@ def nlcd_soils(sjs_result):
     }
 
 
+@shared_task(throws=Exception)
+def us_groundwater_nitrogen(sjs_result):
```

Comment:
Sorry if this is too nitpicky, but I think we can use shorter function names as long as they indicate which datasets are being handled. Prefer `gwn` for Groundwater Nitrogen and `awc` for Average Water-holding Capacity.